lib/pipeline.js

```diff
@@ -135,6 +135,82 @@ Pipeline.prototype = {
   },
 
   /**
+   * Bind to a streams events, and re-emit them on the pipeline
+   * @param {Stream} stream
+   * @param {String|Array} events
+   * @returns {Pipeline}
+   * @chainable
+   */
+  bind( stream, events ) {
```

Comment:
This might be another one of my naive questions, but should all three of these functions check that the stream that's being operated on, is actually a part of the current pipeline?

Or if it works anyway (even if the stream isn't in the pipeline), should there be tests for that?